qt/python/mantidqt/_common.sip

```diff
@@ -224,6 +224,14 @@ public:
   GenericDialog(QWidget *parent = nullptr);
 };
 
+class UserSubWindow : QMainWindow
```

Comment:
Was this required for exporting the help? It should be possible to leave out the `createSubWindow` definition.